### PR TITLE
Added info about what we expect for new projects.

### PR DESCRIPTION
**Pull request by justinabrahms**

### CODE REVIEW

**Comment by pdpinch** [comment]
I've been thinking about this and I feel like the tone is too prescriptive. I understand what "must" and "should" mean, but I think we would be more inviting if we stuck to facts instead of commands. 

For example, instead of 
> You must have a README.

we could say:

> We use a README to describe the repository

I think that allows more room for change, as well as the occasional exception.

**Comment by justinabrahms** [comment]
I was mostly just conforming to https://www.ietf.org/rfc/rfc2119.txt's language. I could make it not be bullet points and instead be long-form with less terse language? Is that the idea?

**Comment by bdero** [comment]
rfc2119 is my favorite song! https://soundcloud.com/mpdehaan/rfc-2119

**Comment by justinabrahms** [comment]
:neckbeard:

**Comment by bdero** [comment]
Props for this document. :+1: Docs like this would have helped me in my onboarding, and I feel learning and discussing these things the hard way can be less than motivating. Practices and processes always exist, the difference is whether they're written down on a transparent discussion medium.

File: new-projects.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+5. You should run tests in continuous-integration.
+6. You should have docker setup for continuous integration and deploymnet.
+7. You should have the relevant linter for your language(s) setup.
+
+READMEs for your project:
+
+1. Must include how to install it.
+2. Must include how to run tests.
+3. Should include relevant build status badges.
+4. Should contain links to relevant documentation.
```
Comment (pwilkins) [inline new-projects.md]: Consider adding, "Should include description of software license."

Comment (justinabrahms) [inline new-projects.md]: Done.

**Comment by justinabrahms** [comment]
Updated to flow a bit better.

**Comment by pdpinch** [comment]
Thanks @justinabrahms this looks good to me.

**Pull request merged**